haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -464,13 +464,13 @@ test_RegistryDAO =
                 , [ FA2.TransferItem { tiFrom = wallet1, tiTxs = [FA2.TransferDestination { tdTo = unTAddress baseDao, tdTokenId = FA2.theTokenId, tdAmount = proposalSize }] } ] -- governance token transfer for freeze
               ]
 
-    , nettestScenarioCaps "checks it can transfer with receive_xtz_entrypoint (#66)" $
+    , testScenario "checks it can transfer with receive_xtz_entrypoint (#66)" $ scenario $
```

Comment:
the test name seems outdated, given that `receive_xtz` was removed

Reply:
Changed.